frontend/src/components/db/SearchableDbSourceTree/treeComponents/DbSourceBrowserTree.tsx

```diff
@@ -35,6 +35,9 @@ const DbSourceBrowserTree = memo(function DbSourceBrowserTree({
         handleTreeChange({ activeTree, setActiveTree, setBreadcrumbs });
       }}
       searchTerm={searchTerm}
+      searchMatch={(node, term) =>
+        node.data.searchField.toLocaleLowerCase().includes(term)
```

Comment:
we are not passing a locale, so I think this will end up using the user's current locale. I'm not sure if this can lead to any inconsistencies / hard to debug bugs if someone with e.g. a Chinese locale starts typing? Maybe safer to use `toLowerCase`, which will return the same string not depending on the users' browser settings?